packs/spell-effects/spell-effect-fortissimo-composition.json

```
@@ -0,0 +1,96 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 4
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
+                        "value": "3"
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.success",
+                        "value": "2"
+                    }
+                ],
+                "flag": "fortissimoBonus",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess"
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "upgrade",
+                "predicate": [
+                    {
+                        "and": [
+                            "bonus:type:status",
+                            {
+                                "or": [
+                                    {
+                                        "and": [
+                                            "bonus:slug:courageous-anthem",
+                                            "courageous-anthem:origin:signature:{item|origin.signature}"
+                                        ]
+                                    },
+                                    {
+                                        "and": [
+                                            "bonus:slug:rallying-anthem",
+                                            "rallying-anthem:origin:signature:{item|origin.signature}"
+                                        ]
+                                    },
+                                    {
+                                        "and": [
+                                            "bonus:slug:song-of-strength",
+                                            "song-of-strength:origin:signature:{item|origin.signature}"
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ],
+                "selectors": [
+                    "ac",
+                    "saving-throw",
+                    "attack-roll",
+                    "damage",
+                    "athletics",
+                    "fortitude",
+                    "reflex"
+                ],
+                "value": "{item|flags.pf2e.rulesSelections.fortissimoBonus}"
```

```suggestion
                "value": "@item.flags.pf2e.rulesSelections.fortissimoBonus"
```
Using the `@` format because we are referencing a number.